File: inflate/tests/langs/php/box.php
<?php

class Box {
    private $size = 1;

    function helper($value) {
        return $value + $this->size;
    }

    function greet($value) {
        return $this->helper($value) * 2;
    }

    function runner() {
        return $this->greet(3);
    }
}

File: inflate/tests/langs/swift/box.swift
class Box {
    var size = 1

    func helper(_ value: Int) -> Int {
        return value + size
    }

    func greet(_ value: Int) -> Int {
        return helper(value) * 2
    }

    func runner() -> Int {
        return greet(3)
    }
}
